pkgs/tools/misc/fsrx/default.nix

```diff
@@ -2,16 +2,16 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "fsrx";
-  version = "1.0.0";
+  version = "1.0.2";
 
   src = fetchFromGitHub {
     owner = "coloradocolby";
     repo = pname;
     rev = "v${version}";
-    sha256 = "sha256-pKdxYO8Rhck3UYxqiWHDlrlPS4cAPe5jLUu5Dajop/k=";
+    sha256 = "sha256-hzfpjunP20WCt3erYu7AO7A3nz+UMKdFzWUA5jASbVA=";
   };
 
-  cargoSha256 = "sha256-5h+ou9FLCG/WWMEQPsCTa1q+PovxUJs+6lzQ0L2bKIs=";
+  cargoSha256 = "sha256-bRStmz2sJnhCKpvoQfc/ZP9B2ZdNHYNEHd+wZErdS1Y=";
 
   passthru = {
     updateScript = gitUpdater {
```

Comment:
Please don't use gitUpdater for rustPlatform.buildRustPackage, it doesn't handle cargoSha256 and will prevent r-ryantm from opening a correct PR for this. https://r.ryantm.com/log/updatescript/fsrx/2022-07-14.log

Reply:
Oh, thanks for the info, I removed the updater. Would something like nix-update-script be better?

Reply:
I guess nix-update-script is okay for this :thinking: You can try revert this bump locally, add nix-update-script as an update script and see if it can produce the change made here with `nix-shell maintainers/scripts/update.nix --argstr package fsrx`